golang/x/relay/types/querier.go

```diff
@@ -184,3 +187,18 @@ func (r QueryResCheckRequests) String() string {
 	json, _ := json.Marshal(r)
 	return fmt.Sprint(string(json))
 }
+
+type QueryParamsCheckProof struct {
```

Comment:
can you add a short description for each of these in the go format? `// QueryParamsCheckProof ________`